packages/openapi3/src/openapi.ts

```diff
@@ -218,42 +219,47 @@ function createOAPIEmitter(program: Program, options: OpenAPIEmitterOptions) {
 
   async function emitOpenAPI() {
     try {
-      for (let route of getRoutes(program)) {
-        if (route.kind !== "Namespace" && route.kind !== "Interface") {
-          reportDiagnostic(program, {
-            code: "resource-namespace",
-            target: route,
-          });
-          continue;
-        }
+      const routes = getRoutes(program);
+      const versions = getVersions(program, routes[0]) ?? [undefined];
+      for (const [index, version] of versions.entries()) {
```

Comment:
should we have a way to emit a single version only?

Reply:
I think so! I wonder if maybe the versioning library can expose that in some way so emitters don't even need to check this at all?